networking/about-contour-operator.adoc

```diff
@@ -0,0 +1,22 @@
+[id="about-contour-operator"]
+= About the Contour Operator
+include::modules/common-attributes.adoc[]
+:context: about-contour-operator
+
+toc::[]
+
+//Dev Preview. OKD only.
+
+The Contour Operator can be installed on {product-title} to manage Contour, an Ingress Controller that supports Gateway API, which exposes Kubernetes resources such as services to external consumers.
```

Comment:
s/Gateway API/the Gateway API

Reply:
Do you need the comma here: _, which exposes_? It seems to make _ an Ingress Controller that supports Gateway API_ a parenthetical phrase where _which exposes_ applies to Contour. 

_ an Ingress Controller that supports Gateway API, an open-source project that exposes Kubernetes resources such as services to external consumers._ ??